plugins/colordialog/dialogs/colordialog.js

```diff
@@ -276,6 +276,22 @@ CKEDITOR.dialog.add( 'colordialog', function( editor ) {
 		title: lang.title,
 		minWidth: 360,
 		minHeight: 220,
+		onShow: function( evt ) {
+			if ( evt.data.colordialog && evt.data.colordialog.selectionColor ) {
+				var selectionColor = evt.data.colordialog.selectionColor;
+				var colors = evt.sender.parts.contents.getElementsByTag( 'td' );
```

Comment:
Please join these declarations to form one:

```javascript
var selectionColor = evt.data.colordialog.selectionColor,
	colors = evt.sender.parts.contents.getElementsByTag( 'td' );
```

Reply:
Additionally, instead of `evt.sender` you can use `this` (as dialog's events are usually called in dialog's context).